client/src/components/aggregations/RichTextWidget.js

```diff
@@ -0,0 +1,43 @@
+import {
+  aggregationWidgetDefaultProps,
+  aggregationWidgetPropTypes
+} from "components/aggregations/utils"
+import _isEmpty from "lodash/isEmpty"
+import React, { useState } from "react"
+import { Button, Collapse, Table } from "react-bootstrap"
+import utils from "utils"
+
+const RichTextWidget = ({ values, whenUnspecified, ...otherWidgetProps }) => {
+  const [showValues, setShowValues] = useState(false)
+  const filteredValues = values.filter(value => !utils.isNullOrUndefined(value))
+  if (_isEmpty(filteredValues)) {
+    return whenUnspecified
+  }
+  return (
+    <div>
+      <Button
+        className="toggle-section-button"
+        style={{ marginBottom: "1rem" }}
+        onClick={() => setShowValues(!showValues)}
+        id="toggleShowValues"
+      >
+        {showValues ? "Hide" : "Show"} {filteredValues.length} values
```

Comment:
Displays: "Show 1 values" (or 'Hide') when there's only one value. Suggestion to make a util method that uses `import pluralize from "pluralize"` and pluralizes "value" based on length, and use that here and in `client/src/components/aggregations/DefaultAggWidget.js`.